server/routes/auth.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const router = express.Router();
+const User = require("../models/User");
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
+
+function validateRegistration(request) {
+  if (request.username.trim().length === 0) {
+    return "Please enter a username";
+  }
+  if (request.password.trim().length < 6) {
+    return "Password must be at least 6 characters long";
+  }
+  // Use if repeat password verificatin is required
+  // if(request.password !== request.repeatPassword){
+  //     return "Passwords don't match"
+  // }
```

Comment:
This one would be done on the frontend only. Backend would receive a single password